student-store-ui/src/components/App/App.jsx

```diff
@@ -1,21 +1,48 @@
 import * as React from "react"
-import { BrowserRouter } from 'react-router-dom'
+import axios from 'axios'
+import { BrowserRouter, Routes, Route} from 'react-router-dom'
 import Navbar from "../Navbar/Navbar"
 import Sidebar from "../Sidebar/Sidebar"
 import Home from "../Home/Home"
+import { useState } from "react"
+import { useEffect } from "react"
 import "./App.css"
+import Productfacts from "../Productdetails/Productdetails"
+//MOST STYLING WAS DONE IN APP.CSS
 
 export default function App() {
+
+  //Shopping cart useState
+  const[ShoppingList, SetShoppingList] = useState([])
```

Comment:
`shoppingList` and `setShoppingList` should not be capitalized, as it makes it looks like these are components